### DISCUSSION TITLE
Running out of spawn tickets

### DISCUSSION THREAD

**Discussion by XDSenDX** [Vdauphin/HeartsAndMinds#1575] [Question and answer] [answered]
Does the game ends when all tickets ran out? Is there any way to start a new one?

Thanks.

**Comment by XDSenDX** [comment]
For me game still continues, it says respawn disabled. I had some players joining and i think their characters are still alive. Not sure if it doesn't count this.

**Comment by Vdauphin** [comment, marked as answer]
Hello,

> Does the game ends when all tickets ran out? 

Yes see http://vdauphin.github.io/HeartsAndMinds/InGame-documentation#respawn-tickets

> Is there any way to start a new one?

New game? In debug console exec server side: `[] call btc_db_fnc_delete;`
Or delete the server profile

Cheers

**Reply by XDSenDX** [reply]
Thanks.
